package.json

```diff
@@ -173,6 +175,7 @@
     "globals": {
       "AVAILABLE_STAGING_LOCALES": null
     },
+    "testEnvironment": "jsdom",
```

Comment:
The new default is "node" (previously the default as "jsdom").

Reply:
oh, interesting.